automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataUseAwareSearchSpec.scala

```diff
@@ -52,6 +58,34 @@ class DataUseAwareSearchSpec extends FreeSpec with WebBrowserSpec with UserFixtu
         Label("commercial-tag").isVisible shouldBe false // We didn't select this one
       }
     }
+
+    "The ontology autocomplete exists and works" in withWebDriver { implicit driver =>
+      val user = UserPool.chooseAnyUser
+
+      withSignIn(user) { _ =>
+        val page = new DataLibraryPage().open
+
+        page.openResearchPurposeModal()
+
+        page.selectRPCheckbox("disease-focused-research")
+
+        SearchField("ontology-autosuggest").setText("fatal")
+
+        val suggestionId = "suggestion-http://purl.obolibrary.org/obo/DOID_0050433"  // fatal familial insomnia
+        val tagId = "doid:0050433-tag"
+
+        await enabled testId(suggestionId) // has a timeout so test will not hang if suggestion never shows
+
+        val ffiSuggestion = Button(suggestionId)
+        ffiSuggestion.isVisible shouldBe true
+
+        ffiSuggestion.doClick()
+
+        await enabled testId(tagId)
+
+        Label(tagId).isVisible shouldBe true
+      }
+    }
```

Comment:
similar comment on this test - after typing fatal, clicking on fatal familial insomnia, and seeing its tag ... try typing another term (this time maybe a substring instead of a whole word) and selecting a second disease to make sure the dropdown works for multiples?

Reply:
👍 Done.